src/authenticated/member/MemberProfileEditPage.js

```diff
@@ -32,15 +34,23 @@ const updateUserQuery = gql`
         id
         name
       }
+      projectsChampioned {
+        id
+        name
+      }
     }
   }
 `;
 
 function prepUserForUpdate(updatedUser) {
   const skillsIds = updatedUser.skills.map(skill => skill.id);
+  const projectsChampionedIds = updatedUser.projectsChampioned.map(
+    project => project.id
+  );
   return {
     ...updatedUser,
-    skillsIds
+    skillsIds,
+    projectsChampionedIds
   };
 }
 
```

Comment:
Similarly, we want the subsequent `updateUserQuery` graphql HOC to refetch `getProject` since those queries could now have stale data after this mutation happens i.e.

```js
graphql(updateUserQuery, {
// ..
  options: {
    refetchQueries: ['getProject']
  }
})
```

I suppose another option would be to use Apollo's APIs to directly update the cached client-side data